consensus/types/presets/gnosis/electra.yaml

```diff
@@ -0,0 +1,3 @@
+# Gnosis preset - Electra
+
+ELECTRA_PLACEHOLDER: 0
```

Comment:
Can this un-spec'ed variable cause issues? why is it necessary to define these files?

Reply:
It's not strictly necessary to define them now, but as soon as someone needs to add variables to the spec they will need all the associated boilerplate (various `ConfigAndPreset` structs and associated functions).

I don't believe there is any danger in having this defined now. The worst thing is that `ELECTRA_PLACEHOLDER` will temporarily be included in the response of `/eth/v1/config/spec`. It's a little messy but it should be safe given that we have Deneb variables present in this endpoint already even though mainnet has not forked.

Happy to hear alternate opinions of course!